vscode-powershell.build.ps1

```diff
@@ -145,15 +137,15 @@ task Package UpdateReadme, {
         throw "Unable to find PowerShell EditorServices"
     }
 
-    Write-Host "`n### Packaging PowerShell-insiders.vsix`n" -ForegroundColor Green
+    $packageName = "$($script:PackageJson.name)-$($script:PackageJson.version).vsix"
```

Comment:
@rjmholt Are you ok with `PowerShell` changing to `powershell`? This _is_ the name in the `package.json`.